docs/src/docs/modelkit/premade-modelkits.md

```diff
@@ -0,0 +1,60 @@
+# Pre-Built ModelKits
```

Comment:
Some inconsistency here. We refer to this page as `pre-made modelkits` but the title is Pre-Built. I prefer pre-built to be used out of the 2 but pre-packaged is another term we can consider.